lib/gems/pending/util/vmdb-logger.rb

```diff
@@ -108,9 +108,13 @@ def self.log_hashes(logger, h, options = {})
     filter = Array(options[:filter]).flatten.compact.map(&:to_s) << "password"
     filter.uniq!
 
-    YAML.dump(h).split("\n").each do |l|
+    values = YAML.dump(h).gsub(MiqPassword::REGEXP, "[FILTERED]")
+    values.split("\n").each do |l|
       next if l[0...3] == '---'
-      logger.send(level, "  #{l}") unless filter.any? { |f| l.include?(f) }
+      if key = filter.detect { |f| l.include?(f) }
```

Comment:
@lfu Can you fix the bot warning...to solve it, just use parens around your if conditional...

```ruby
if (key = filter.detect { |f| l.include?(f) })